package.json

```diff
@@ -48,23 +48,23 @@
     "node": ">=16"
   },
   "scripts": {
-    "prepare": "node npm-scripts.js prepare",
-    "postinstall": "node npm-scripts.js postinstall",
-    "typescript:build": "node npm-scripts.js typescript:build",
-    "typescript:watch": "node npm-scripts.js typescript:watch",
-    "worker:build": "node npm-scripts.js worker:build",
+    "coverage:node": "node npm-scripts.js coverage:node",
+    "format:worker": "node npm-scripts.js format:worker",
+    "install-clang-tools": "node npm-scripts.js install-clang-tools",
+    "install-deps:node": "node npm-scripts.js install-deps:node",
     "lint": "npm run lint:node && npm run lint:worker",
     "lint:node": "node npm-scripts.js lint:node",
     "lint:worker": "node npm-scripts.js lint:worker",
-    "format:worker": "node npm-scripts.js format:worker",
+    "postinstall": "node npm-scripts.js postinstall",
+    "prepare": "node npm-scripts.js prepare",
+    "release": "node npm-scripts.js release",
+    "release:check": "node npm-scripts.js release:check",
     "test": "npm run test:node && npm run test:worker",
     "test:node": "node npm-scripts.js test:node",
     "test:worker": "node npm-scripts.js test:worker",
-    "coverage:node": "node npm-scripts.js coverage:node",
-    "install-deps:node": "node npm-scripts.js install-deps:node",
-    "install-clang-tools": "node npm-scripts.js install-clang-tools",
-    "release:check": "node npm-scripts.js release:check",
-    "release": "node npm-scripts.js release"
+    "typescript:build": "node npm-scripts.js typescript:build",
+    "typescript:watch": "node npm-scripts.js typescript:watch",
+    "worker:build": "node npm-scripts.js worker:build"
```

Comment:
Definitely I don't like these changes. Order of scripts was set on purpose. I don't want that those are ordered alphabetically. Let's revert this, please.

Reply:
It was the same, to easily identify changes, but ok, I'll revert it.

Reply:
Thanks.

Reply:
Done.